### ISSUE TITLE
Model Request  #2

### ISSUE THREAD

**Issue by Divetoxx** [gafr/chainer-fast-neuralstyle-models#15]
Hi there!
Could anyone please train a model using this style image and with --image_size 1024 ?  
![mermaid_style_1024](https://cloud.githubusercontent.com/assets/1689751/22177985/da3f678e-e03b-11e6-9498-4aac21ff171d.png)
I think the resulting pictures could be beautiful. I tried to use this style with this online service: https://dreamscopeapp.com/ and got this result:

![mermaid_example](https://cloud.githubusercontent.com/assets/1689751/22178011/95845112-e03c-11e6-8477-308861bae7df.png)

Sad that this service gives only 512x512 results :( I think 1024x1024 would be more beautiful...
Could anyone please help? I have no powerful video card :(

**Comment by 6o6o** [comment]
Training on that size would require > 12GB of GPU memory. Even the new Titan X has only 12 GB. If you're lucky, you may be able to squeeze into it, otherwise Tesla P is your only hope. Good luck )

**Comment by Divetoxx** [comment]
Oh, could you forgive me please? I was too grredy.
Let it be 512.
And may I ask, why don't I see tons of contributed models? I have no powerful GPU but many people have...

**Comment by 6o6o** [comment]
There are tons of similar projects, each with their own models collection. Here's a few worth checking out:

https://github.com/lengstrom/fast-style-transfer
https://github.com/jcjohnson/fast-neural-style

Regarding this one, I have about a dozen of models lying around **BUT** they were trained months ago, and recently the author upgraded his implementation with [resize-convolution](/yusuketomoto/chainer-fast-neuralstyle/tree/resize-conv) technique, which produces far superior results. This makes the old models obsolete and kinda useless, since the upgrade is not backwards-compatible.

**Comment by Divetoxx** [comment]
Thanks! It's very interesting! May I ask question? I see yusuketomoto has upgraded his implementation a month ago. But your https://github.com/6o6o/chainer-fast-neuralstyle I see was updated last time 5 months ago - so still we can use old models?
And I don't see any word about flow technique on yusuketomoto's page... It's sad...

**Comment by 6o6o** [comment]
Yes, my fork is based on previous version of the code, so they are compatible. Actually, it is still the current master branch of the original project, (resize-conv is a separate branch), so they are compatible with original as well. By saying _old_, I meant all the models in this repo, including latest, sorry for misleading.

Optical flow is an optional technique to smooth out motion rendering by partially blending frames together. It applies only to sequence of images, therefor not available in the original - it deals only with single ones.
